Automattic/wp-calypso
# Inline help: Remove onboarding welcome

#### Changes proposed in this Pull Request

<img width="415" alt="51885784-32e51b00-23e1-11e9-976c-9f25fe325ea9" src="https://user-images.githubusercontent.com/1270189/78924793-3ea18800-7a4f-11ea-90b8-095648c49d76.png">

Seems we no longer support the onboarding welcome message we added to the inline help in #30406, so this PR removes all the related code in order to clean up the codebase from dead code.

#### Testing instructions

- Create a new site.
- Make sure there are no regressions in the inline help and it looks exactly the same as production.

Part of #40646
